packages/utils/src/lib/reports/sorting.ts

```diff
@@ -60,7 +60,7 @@ export function sortReport(report: ScoredReport): ScoredReport {
       ...audit,
```

Comment:
The line above (can't click it) - `plugin.audits.sort(compareAudits)` - why does that not get flagged? It mutates `plugin` :sweat_smile:

Reply:
Because `immutable-data` is not enabled yet.
Either way, fixed.